kvdb-rocksdb/src/lib.rs

```diff
@@ -596,6 +596,22 @@ impl Database {
 			.unwrap_or(0)
 	}
 
+	/// The number of keys in a column (estimated).
+	/// Does not take into account the data in the overlay.
```

Comment:
I think "overlay" is an internal implementation detail, we should call it "unflushed" maybe.